pkgs/applications/networking/instant-messengers/hangups/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, python3 }:
+let 
+  packageOverrides = self: super:{
+
+    ConfigArgParse = super.ConfigArgParse.overridePythonAttrs (oldAttrs: rec {
+      version = "0.11.0";
+      src = oldAttrs.src.override {
+        inherit version; 
+        sha256 = "12dcl0wdsjxgphxadyz9bdzbvlwfaqgvba9s59ghajw4yqiyi2kc";
+      };
+    });
+
+    aiohttp = super.aiohttp.overridePythonAttrs (oldAttrs: rec {
+      version = "2.3.10";
+      src = oldAttrs.src.override {
+        inherit version;
+        sha256 = "0r0955ar0yiwbp3war35z8zncs01nq84wdwk0v3s8f547dcadpca";
+      };
+    }); 
+
+    MechanicalSoup = super.MechanicalSoup.overridePythonAttrs (oldAttrs: rec {
+      version = "0.6.0";
+      src = oldAttrs.src.override {
+        inherit version;
+        extension = "zip";
+        sha256 = "0ka3nximli7wcpy381grn6d75fnpf91050iwdcj4shf53z0m1fg2";
+      };
+    }); 
+  };
+  pythonPackages = (python3.override { inherit packageOverrides; }).pkgs;
+
+in pythonPackages.buildPythonApplication rec {
+  pname = "hangups";
+  version = "0.4.4";
+  disabled = !pythonPackages.isPy3k;
+
+  propagatedBuildInputs = with pythonPackages; [ 
+    ConfigArgParse 
+    requests
+    aiohttp
+    ReParser
+    appdirs
+    urwid
+    protobuf3_1
```

Comment:
Please use `protobuf` and pin the version as for the packages above. You'll have to use
```
protobuf = super.protobuf.override {
  protobuf = pkgs.protobuf3_1;
};
```
however.